ppr-ui/src/utils/validation-helper.ts

```diff
@@ -173,10 +173,7 @@ export function validateSearchRealTime (searchState: partialSearchState): Search
     case UISearchTypes.AIRCRAFT:
       searchValue = searchValue?.replace(dash, '')
       if (searchValue?.length > 25) {
-        validation.searchValue.message = 'Maximum 25 letters'
-      }
-      if (!lettersOnly.test(searchValue)) {
```

Comment:
Just need to add the special char check here instead of letters only

Reply:
Patrick verified special chars allowed